_tut/docs/2x/eval/callback.md

```diff
@@ -7,8 +7,6 @@ description: |
   A listener type that can be called asynchronously with the result of a computation. Used by the Monix Task.
     
 tut:
-  scala: 2.12.4
-  binaryScala: "2.12"
```

Comment:
Something is probably wrong with the merge, because these lines shouldn't be removed on master — I added them back.

Reply:
Yeah there should be only 1 changed file